actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -53,11 +53,6 @@ def initialize(view_context)
           @view_context = view_context
         end
 
-        # Transforms a Hash into HTML Attributes, ready to be interpolated into
-        # ERB.
-        #
-        #   <input <%= tag.attributes(type: :text, aria: { label: "Search" }) %> >
-        #   # => <input type="text" aria-label="Search">
```

Comment:
I wonder if this method was public or meant to be public at one point? 🤔 

The other thought I have, it's perfectly fine and normal to have comments for internal use, as long as they are accurate. I've seen this a lot in AR, for example.

Reply:
It certainly is intended for public use. This commit originally replaced the comment with the `:method: tag.attributes` header comment in hopes of making the method more discoverable without duplicating the content.

I like your idea of retaining the original comment, though. I've added it back.